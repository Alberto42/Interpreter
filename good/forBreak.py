y = 0
for x in range(0,9) {
    y = x
    if x > 4 {
        break
    }
}