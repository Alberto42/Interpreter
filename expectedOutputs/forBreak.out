Right (OK,fromList [(Ident "y",VInt 5)])
